packages/mint-components/src/components/tax-and-cash/sqm-indirect-tax-form/sqm-indirect-tax-form.feature

```diff
@@ -0,0 +1,81 @@
+@owner:andy @author:andy
+Feature: Indirect Tax Form
+
+  Background: A user has submitted their personal information in Tax Form Step One
+    Given a user is on the Indirect Tax Form
+
+  @minutia
+  Scenario Outline: Different indirect tax inputs are shown depending on the country of a participant
+    When <option> is selected based on based the participant <country> from step 1
+    Then different <inputs> appear
+
+    Examples: 
+      | country        | option                                                           | inputs                        |
+      | Canada         | I am registered for HST in Canada                                | Province, Indirect Tax Number |
+      | United States  | I am registered for Indirect Tax in a different Country / Region | Country, VAT Number           |
```

Comment:
The wording of VAT is a bit confusing here in general, @Locrian24 what does the Impact frontend do today? Do they just call it VAT? Or is the field label conditional on the country selected?